scripts/bump-oss-version.js

```diff
@@ -76,5 +76,18 @@ if (exec(`git tag v${version}`).code) {
   exit(1);
 }
 
+// Push newly created tag
+exec(`git push origin v${version}`);
+
+// Tag latest if doing stable release
+if (version.indexOf('rc') === -1) {
```

Comment:
it was an insane idea to use ` ` for all strings.
But let's keep it consistently insane in this file :)
Until you are ready to migrate all of the non template strings to normal quotes